views/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
```

Comment:
so looks like nunjucks is set up -> https://mozilla.github.io/nunjucks/templating.html

take a look at that link. It talks about template inheritance and how we can reuse templates

you should be able to create some sort of file aka base.njk

which says something like (excuse my sudo code)

```
<header stuff>

<main>
{ % block content %} {% endblock % }
</main>

<footer stuff>
```
and then use this like

```
{% extends "base.njk" %}


{% block content %}

<the stuff that thomas is working on / any future pages>

{% endblock %}
```

Now you have a reusable template which we can put on all the pages :)